specification/botservice/resource-manager/Microsoft.BotService/preview/2021-05-01-preview/botservice.json

```diff
@@ -1609,6 +1609,10 @@
           "type": "boolean",
           "description": "Whether the bot is in an isolated network"
         },
+        "disableLocalAuth": {
+          "type": "boolean",
```

Comment:
>boolean [](http://example.com/codeflow?start=19&length=7)

ARM guidance is to discourage booleans in favor of string enums, e.g. localAuthType { Required, NotRequired }. These are usually easier for customers to understand and use, are extensible to more than two values, and can save complexity by eliminating invalid combinations of multiple booleans. This isn't a requirement, but you should consider it.

Reply:
That is the type expected of that property. Just reusing as is per the guiding doc.

Reply:
That is the type expected of that property. Just reusing as is per the guiding doc.